Sources/ArcGISToolkit/Components/Popups/Attachments/AttachmentsPopupElementModel.swift

```diff
@@ -58,6 +58,8 @@ import QuickLook
             defaultSystemName = "film"
         case .document, .other:
             defaultSystemName = "doc"
+        @unknown default:
+            defaultSystemName = nil
```

Comment:
Have `defaultSystemName == nil` means that the user set their own thumbnail, so a non-nil value is required.  A "question mark" should denote that something is amiss...

```suggestion
            defaultSystemName = "questionmark"
```